src/lib/coda_base/transaction_logic.ml

```diff
@@ -302,19 +377,33 @@ module Make (L : Ledger_intf) : S with type ledger := L.t = struct
         {user_command; previous_receipt_chain_hash= account.receipt_chain_hash}
       in
       let%bind () = validate_nonces nonce account.nonce in
+      (* TODO: Put actual value here *)
+      let current_global_slot = Global_slot.zero in
```

Comment:
can you open an issue for this  TODO?

Reply:
Done, and added comments referencing the issue (#4036).